package.json

```diff
@@ -35,7 +35,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "CI= react-scripts build",

```

Comment:
Is this needed? I changed this while testing out netlify.

Reply:
what does this line do?

Reply:
When I was originally trying to get netlify working, I found this fix. Apparently it prevents warnings from creating a fatal error in the build process. This [link](https://www.reddit.com/r/reactjs/comments/hlqcqm/app_suddenly_fails_to_build_on_netlify_but_runs/) explains it some.